### DISCUSSION TITLE
Update parametre group for a ElastiCache (redis) cluster via CDK

### DISCUSSION THREAD

**Discussion by vinodjayachandran** [aws/aws-cdk#24977] [Q&A]
I would like to close idle db connections to redis. `timeout` parameter let's you do that and [CfnParametreGroup](https://docs.aws.amazon.com/cdk/api/v1/python/aws_cdk.aws_elasticache/CfnParameterGroup.html) seems to be the way for it.

But my Redis Cluster is created via [CfnReplicationGroup](https://docs.aws.amazon.com/cdk/api/v1/python/aws_cdk.aws_elasticache/CfnReplicationGroup.html) . How can i associate **CfnParametreGroup** to **CfnReplicationGroup** ? There seems to be no way from the docs.

Looking for guidance on what's the correct procedure to update parameters for redis cluster via CDK ?

**Comment by peterwoodworth** [comment]
Doesn't `cache_parameter_group_name` create the association between the parametergroup and replicationgroup?

**Reply by acosta-edgar** [reply]
The `cache_parameter_group_name` property is available on CfnReplicationGroup. However, it is not clear how to get the name of the CfnParemeterGroup, it is not a property.
